test/framework/math/quadrature/quadrature_test1.lua

```diff
@@ -23,8 +25,8 @@ print("weights:")
 PrintTable(qdata.weights, 2)
 print()
 
---################################################
-q = squad.GaussChebyshevQuadrature.Create({N = 4, verbose = true})
+-- 
```

Comment:
Looks like a trailing white space. It seems the `StyLua` piggy-backs on the `.editorconfig` (which we have). But for some reason it ignored our request to trim trailing white spaces. Can you try to add:
```
trim_trailing_whitespace = true
```
and see of that fixes the problem?

Reply:
Interesting that the whitespace diff it is not showing here, but it does when looking at `Changed files`...

Reply:
It's strange it only show up in `Changed files`.  I believe this was my fault. I did some editing after running `stylua` to remove the `#####...` lines that probably introduced the added white space. I've cleaned that up.